src/components/docs/WaveBackground/styles.module.css

```diff
@@ -0,0 +1,22 @@
+.root {
+  margin-left: calc( -1 * var(--doc-padding-left) );
+  position: relative;
+  width: calc( 100% + var(--doc-padding-left) + var(--doc-padding-right) );
+  z-index: -1;
```

Comment:
This needs to be moved to cut the middle tab in the middle on the left side. `top: -35px;`